src/components/3-organisms/SetSearch.jsx

```diff
@@ -137,7 +136,7 @@ function SetSearch(props) {
         >
             <AccordionSummary
                 expandIcon={<ExpandMoreIcon />}
-                className={classes.label}
+                sx={styles.label(true)}
```

Comment:
As commented before, that would be better to not use logic parameters for styles. When reading this line, it makes no sense.